python/src/iceberg/utils/datetime.py

```diff
@@ -77,6 +89,12 @@ def timestamptz_to_micros(timestamptz_str: str) -> int:
     raise ValueError(f"Invalid timestamp with zone: {timestamptz_str} (must be ISO-8601)")
 
 
+def micros_to_timestamp(micros: int, tzinfo: timezone | None = None):
+    dt = timedelta(microseconds=micros)
+    unix_epoch_datetime = datetime(1970, 1, 1, 0, 0, 0, 0, tzinfo=tzinfo)
```

Comment:
Can you use `EPOCH_TIMESTAMP` and `EPOCH_TIMESTAMPTZ` instead of creating a new `datetime` here?

```python
def micros_to_timestamp(micros: int):
    return EPOCH_TIMESTAMP + timedelta(microseconds=micros)

def micros_to_timestamptz(micros: int):
    return EPOCH_TIMESTAMPTZ + timedelta(microseconds=micros)
```